src/objects/aff/zcl_abapgit_json_handler.clas.abap

```diff
@@ -191,4 +207,52 @@ CLASS zcl_abapgit_json_handler IMPLEMENTATION.
     ENDLOOP.
   ENDMETHOD.
 
+
+  METHOD set_defaults.
+
+  ENDMETHOD.
+
+  METHOD set_original_language_deser.
+    DATA:
+      lv_iso_language      TYPE i18_a_langiso2,
+      lv_original_language TYPE sy-langu.
+
+
+    lv_iso_language = co_ajson->get_string( '/header/originalLanguage' ).
+
+    cl_i18n_languages=>locale_to_sap1(
+      EXPORTING
+        iv_localename        = lv_iso_language
+        iv_consider_inactive = abap_true
+      IMPORTING
+        ev_lang_sap1         = lv_original_language
+      EXCEPTIONS
+        no_assignment        = 1
+        OTHERS               = 2
+    ).
+    IF ( sy-subrc <> 0 ).
+      IF ( matches( val = lv_iso_language pcre = `[A-Z]{2}` ) ).
```

Comment:
this is the first statement to fix, `IF foo` does not work in 702, only `IF foo = bar`

and PCRE is a 755 feature